burn-book/src/building-blocks/tensor.md

```diff
@@ -155,7 +155,8 @@ Those operations are available for all tensor kinds: `Int`, `Float`, and `Bool`.
 | `tensor.not_equal(other)`             | `x != y`                             |
 | `tensor.permute(axes)`                | `tensor.permute(axes)`               |
 | `tensor.movedim(src, dst)`            | `tensor.movedim(src, dst)`           |
-| `tensor.repeat(2, 4)`                 | `tensor.repeat([1, 1, 4])`           |
+| `tensor.repeat_dim(2, 4)`             | `tensor.repeat([1, 1, 4])`           |
+| `tensor.repeat(&[(0,2),(2,4)])`       | `tensor.repeat([2, 1, 4])`           |
```

Comment:
To be consistent we should describe params being passed just like other examples in this doc. Also the pytorch equivalent examples should match. For example,  `tensor.repeat(&[(0,2),(2,4)])` is not same as `tensor.repeat([2, 1, 4])`